docs/projects/lottie/layers/linear_gradient.rst

```diff
@@ -0,0 +1,27 @@
+Linear Gradient Layer
+=====================
+
++---------------------+---------------------------------------------------------------------------+
+|       Property      |                         Path to Property in lottie                        |
++=====================+===========================================================================+
+|       Z depth       |               Depends on ordering of layers in lottie format              |
++---------------------+---------------------------------------------------------------------------+
+|        Amount       |   layers/shape.json -> "ef" -> effects/gFill.json -> effects/opacity.json |
++---------------------+---------------------------------------------------------------------------+
+|     Blend_method    |Partially supported                                                        |
++---------------------+---------------------------------------------------------------------------+
+|        Gradient     |    layers/shape.json -> "ef" -> effects/gfill.json                        |
++---------------------+---------------------------------------------------------------------------+
+|       p1            |layers/shape.json ->  "ef" -> effects/gFill.json ->  "s"                   |
++---------------------+---------------------------------------------------------------------------+
+|       p2            |layers/shape.json ->  "ef" -> effects/gFill.json ->  "e"                   |
++---------------------+---------------------------------------------------------------------------+
+|      Loop           |                               Not supported                               |
++---------------------+---------------------------------------------------------------------------+
+|      Zigzag         |                               Not supported                               |
++---------------------+---------------------------------------------------------------------------+
+
+Important points
+----------------
+
+- p1 and p2 parameters are used to calculate the gradient start point -> "s" and the gradient end point -> "e" respectively.
```

Comment:
Please make a note about `point1` and `point2` also, which you introduced for the rectangle layer.